docs/src/references/agent/device-management-api.md

```diff
@@ -2,11 +2,12 @@
 title: Device Management API
 tags: [Reference, Agent, API]
 sidebar_position: 2
+description: Details of the Device Management API
 ---
 
 # Device Management API
 
-The agent and the operation plugins enable device management on thin-edge,
+The agent and the operation plugins enable device management on %%te%%,
```

Comment:
Oh, I was wrong! I though that the description was used for the descriptions given in the tag pages.

But apparently this is not the case. The page for the tag _Agent_ is currently displayed as:

> __The Agent__
The agent and the operation plugins are the main components used by %%te%%

Reply:
The description field is used in the DocCardList page preview and it is also added to the page's meta data, https://docusaurus.io/docs/markdown-features/head-metadata#markdown-page-description

Reply:
Ah wait, do you mean that "tag page" which is displayed when you click on one of the tags?

Reply:
I found the tags page you were referring to (I don't use that page often so it was overlooked).

Resolved by c8ea100c5cc37dd185f5cc3be9e144ce145874a7